generators/client/templates/react/src/main/webapp/app/shared/layout/header/header.tsx

```diff
@@ -9,7 +9,7 @@ import {
 import {
   FaHome, FaThList, FaUserPlus, FaUser, FaFlag, FaHeart,
   FaList, FaTasks, FaDashboard, FaBook, FaWrench, FaSignIn, FaSignOut,
-  FaClockO, FaHddO,
+  FaClockO, FaHddO, FaBell,
```

Comment:
add this after the // tslint:disable-next-line so that this doesn't fail when choosing Cassandra